### ISSUE TITLE
Teleporting inside dungeon while in combat

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5619]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This was fixed somewhere in the summer but for some reason the issue is back, you can teleport inside dungeon while in combat, can be used to escape from fights in world pvp which happens in org/sw mostly
**Expected behaviour**: Tell us what should happen instead
When you are in combat you shouldn't be able to teleport inside a dungeon
**Steps to reproduce the problem**:
Get in a dungeon group, teleport out, get in combat, then teleport in.
1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by RandomXD** [comment]
Here is the first report made by someone and it's marked as fixed but the bug is still present and exploited
https://github.com/dalaranwow/dalaran-wow/issues/5470

**Comment by Golganeth** [comment]
#5629
